certification/internal/service/service.go

```diff
@@ -0,0 +1,5 @@
+package service
+
+type TagLister interface {
```

Comment:
IMO, using a generic name might be enable us to extend this interface for the future. I cannot think of a name that I like enough, but sth along the line of RepositoryService which includes all repo-related operations might be ok.

Reply:
This is idiomatic golang. Small interfaces that are generic. This IMO is a generic name. See all the io interfaces, like Reader, Writer, Closer. They contain one method. But they might all be implemented together.